safe/common/parameters/test/example.py

```diff
@@ -5,7 +5,7 @@
 
 from safe.test.utilities import get_qgis_app
 
-QGIS_APP, CANVAS, IFACE, PARENT = get_qgis_app()
+QGIS_APP, CANVAS, IFACE, PARENT = get_qgis_app(qsetting='InaSAFETest')
```

Comment:
Could you make this 'InaSAFETest' the default string in the function? instead of updating all usages.

Reply:
Hi ! yes of course !